### PR TITLE
Expand terminal menu actions

### PR DESCRIPTION
**Pull request by versecafe**
Release Notes:

- Added `Copy`, `Paste`, `Select All`, & `New Terminal`

image

### CODE REVIEW

**Comment by versecafe** [comment]
This fixes a complaint someone had about the menu only being `Clear` and `Close`, now makes it a full functioning menu covering all common actions and matching VSC minus some irrelevant ones that only apply to it being electron based like `Copy as HTML` or unsupported such as `Split Terminal`

File: crates/terminal_view/src/terminal_view.rs [hunk 1]
```diff
@@ -214,7 +214,13 @@ impl TerminalView {
         cx: &mut ViewContext<Self>,
     ) {
         let context_menu = ContextMenu::build(cx, |menu, _| {
-            menu.action("Clear", Box::new(Clear))
+            menu.action("New Terminal", Box::new(NewTerminal))
+                .separator()
+                .action("Copy", Box::new(Copy))
+                .action("Paste", Box::new(Paste))
+                .action("Select All", Box::new(SelectAll))
```
Comment (SomeoneToIgnore) [inline crates/terminal_view/src/terminal_view.rs:221]: This does not show any key combination as you can see on the screenshot, albeit `cmd-a` already works on mac, and I suspect that `ctrl-a` will work on Linux too.

Let's add corresponding entries into `default-macos.json` and `default-linux.json` into the
```
{
    "context": "Terminal",
...
}
```
blocks: `"cmd-a": "editor::SelectAll",` and a similar one for the Linux version.

**Review by SomeoneToIgnore — COMMENTED** [review]
Thank you, looks good, left a few minor notes.

File: crates/terminal_view/src/terminal_view.rs (hunk 1 above)
Comment (versecafe) [reply crates/terminal_view/src/terminal_view.rs:221]: Taken care of works on MacOS & Linux correctly now, thanks for catching that

File: crates/terminal_view/src/terminal_view.rs [hunk 2]
```diff
@@ -3,7 +3,7 @@ pub mod terminal_element;
 pub mod terminal_panel;
 
 use collections::HashSet;
-use editor::{scroll::Autoscroll, Editor};
+use editor::{actions::SelectAll, scroll::Autoscroll, Editor};
```
[the remark this answers is not in the document]
Comment (versecafe) [reply crates/terminal_view/src/terminal_view.rs:6]: Cleaned

**Comment by versecafe** [comment]
@SomeoneToIgnore Taken care of all notes and tested the changes

**Review by SomeoneToIgnore — APPROVED** [review]
Thanks!

**Pull request merged**